packages/gatsby/src/recipes/recipe-machine.js

```diff
@@ -0,0 +1,214 @@
+const { Machine, assign } = require(`xstate`)
```

Comment:
Why is it a state machine? This seems purely sequential. State machine looks like an overkill.